gnomad_hail/utils/sample_qc.py

```diff
@@ -659,6 +659,47 @@ def run_pca_with_relateds(
         return pca_evals, pca_scores, pca_loadings
 
 
+def annotate_relationship(
+    relatedness_ht: hl.Table,
+    first_degree_threshold: List[float] = [0.1767767, 0.4],
```

Comment:
to continue the streak of only posting naming comments, it's slightly confusing that `first_degree_threshold` is a range and `second_degree_threshold` is a number. Maybe this could be two args like ..._upper_threshold, _lower_threshold.

Reply:
Also, this is definitely a nitpick, but since the threshold argument is a fixed size (2 elements), `Tuple[float, float]` might be a more appropriate type for it.